pkgs/servers/ankisyncd/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3
+, anki
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ankisyncd";
+  version = "2.1.0";
+  src = fetchFromGitHub {
+    owner = "tsudoko";
+    repo = "anki-sync-server";
+    rev = version;
+    sha256 = "6a140afa94fdb1725fed716918875e3d2ad0092cb955136e381c9d826cc4927c";
+  };
+  format = "other";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/${python3.sitePackages}
+
+    cp -r ankisyncd utils ankisyncd.conf $out/${python3.sitePackages}
+    mkdir $out/share
+    cp ankisyncctl.py $out/share/
+
+    runHook postInstall
+  '';
+
+  fixupPhase = with python3; ''
+    PYTHONPATH="$PYTHONPATH:$out/${sitePackages}:${anki}"
+
+    makeWrapper "${interpreter}" "$out/bin/ankisyncd" \
+          --set PYTHONPATH $PYTHONPATH \
+          --add-flags "-m ankisyncd"
+
+    makeWrapper "${interpreter}" "$out/bin/ankisyncdctl" \
```

Comment:
Hmm that's a good point. I think I added it because I found the name `ankisyncctl` to be a bit inconsistent, but you're right that it could be confusing. I will change it.

Reply:
I think it was probably chosen to be consistent with `systemd`/`systemctl` :)

Reply:
Ah that makes sense :-)